readme.md

```diff
@@ -73,98 +77,27 @@ Puppeteer test scripts are in `benchmark/puppeteer`. To run those scenarios (req
 ```bash
 cd benchmark/puppeteer
 npm install
-# Then run each scenario with `node`:
-node homepage-landing.js
-```
-
-### Playwright scenarios for Greenframe
```

Comment:
I tried, and couldn't get Greenframe to run these scenarios to run when preparing for the workshop so I removed them to reduce the number of things to explain during the session.

That said, when introducing folks to this topic, the UX presented by the greenframe tool is really nice to introducing the idea of measuring containerised images directly. I've written a bit more here in this issue, and linked to a sample repo to demonstrate what I mean, and how I'd probably try being new developers along in future.

https://github.com/mrchrisadams/gmt-check-a-website